.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-dist: xenial
+dist: bionic
```

Comment:
I must have missed the reason for switching dists in travis. What was that for?

Reply:
Uff, this slipped through and caused a regression: older GCC (<6) need explicit C++11 flags in the build system, otherwise they will fail to build: #2892